hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieDemo.java

```diff
@@ -140,7 +140,7 @@ public void testParquetDemo() throws Exception {
     testHiveAfterSecondBatchAfterCompaction();
     // testPrestoAfterSecondBatchAfterCompaction();
     // testTrinoAfterSecondBatchAfterCompaction();
-    testIncrementalHiveQueryAfterCompaction();
+    // testIncrementalHiveQueryAfterCompaction();
```

Comment:
Why comment out without JIRA tasks trace?

Reply:
HUDI-8572. I had disabled temporarily. I have another patch stacked on top of this. Either I will fix it or add JIRA in the comment in my PR